index.bs

```diff
@@ -749,26 +816,25 @@ Registry instances are initially created with the internal slots described in th
 1. Return _p_.
 </emu-alg>
 
-<h4 id="request-satisfy" aoid="RequestSatisfy">RequestSatisfy(loader, key)</h4>
+<h4 id="request-satisfy" aoid="RequestSatisfy">RequestSatisfy(entry)</h4>
 
 <emu-alg>
-1. Let _entry_ be EnsureRegistered(_loader_, _key_).
-1. Let _satisfyStageEntry_ be GetStage(_entry_, "satisfy").
+1. Let _satisfyStageEntry_ be GetStage(_entry_.[[Loader]], "satisfy").
 1. If _satisfyStageEntry_ is *undefined*, return a promise resolved with *undefined*.
 1. If _satisfyStageEntry_.[[Result]] is not *undefined*, return _satisfyStageEntry_.[[Result]].
-1. Let _p_ be the result of transforming RequestInstantiate(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
+1. Let _p_ be the result of transforming RequestInstantiate(_entry_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
```

Comment:
Reusing the variable name doesn't really bother me. Lexical scope is lexical scope. :)